stdlib/ballerina-builtin/src/main/java/org/ballerinalang/nativeimpl/socket/SocketConstants.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.ballerinalang.nativeimpl.socket;
+
+/**
+ * Constant variable for socket related operations.
+ */
+public class SocketConstants {
+
+    public static final String PKCS_STORE_TYPE = "PKCS12";
```

Comment:
Shall we specify java Docs explaining what each of these constants do ?

Reply:
I think constant names are self-explanatory.